app/src/main/java/org/oppia/app/walkthrough/welcome/WalkthroughWelcomeViewModel.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.app.walkthrough.welcome
+
+import org.oppia.app.viewmodel.ObservableViewModel
+
+/** [ViewModel] for displaying User profile details in walkthrough Welcom Fragment. */
+class WalkthroughWelcomeViewModel : ObservableViewModel() {
+  var profileName: String = ""
```

Comment:
Use observable field like this:
`val profileName = ObservableField<String>("")`

Reason: In current code the `profileName` is set once and is connected to the xml. Now even if you update the `profileName` it will not update the xml file because it is not observing.

Now once you make it observable whenever the value changes it will notify the xml file that the value has been updated and will be reflected in the xml file.

Reply:
Oh yes right